rs/nns/governance/src/governance/tla/claim_neuron.rs

```diff
@@ -0,0 +1,103 @@
+use std::collections::{BTreeMap, BTreeSet};
+use tla_instrumentation::{
+    Label, ResolvedStatePair, TlaConstantAssignment, TlaValue, ToTla, Update, VarAssignment,
+};
+
+use super::common::{default_account, function_domain_union, governance_account_id};
+
+pub fn claim_neuron_desc() -> Update {
+    const PID: &str = "Claim_Neuron";
+    let default_locals = VarAssignment::new()
+        .add("account", default_account())
+        .add("neuron_id", 0_u64.to_tla_value());
+
+    Update {
+        default_start_locals: default_locals.clone(),
+        default_end_locals: default_locals,
+        start_label: Label::new("ClaimNeuron1"),
+        end_label: Label::new("Done"),
+        process_id: PID.to_string(),
+        canister_name: "governance".to_string(),
+        post_process: |trace| {
+            let constants = extract_claim_neuron_constants(PID, trace);
+            post_process_trace(trace);
+            constants
+        },
+    }
+}
+
+fn extract_claim_neuron_constants(pid: &str, trace: &[ResolvedStatePair]) -> TlaConstantAssignment {
+    let constants = BTreeMap::from([
```

Comment:
It's actually slightly different. I have "Claim_Neuron_Process_Ids" instead of "Split_..." and I do not have "Minting_Account_Id".

How about I extract a function for the common parts? Would you want to include Minting_Account_Id? I think I would need to add it to my model, which is not a problem.

Reply:
Extracted common parts into `mod.rs`.